tests/clims/models/test_container.py

```diff
@@ -133,6 +143,21 @@ def test_can_output_default_string_info(self):
         # Expecting a digit followed by some empty columns:
         assert re.match(r"^\d+[ |].+", first_line) is not None
 
+    @override_settings(DEBUG=True)
+    def test_get_content_incurs_no_extra_call(self):
+        # Getting the contents from a container incurs no extra calls to the DB if called correctly
+
+        from django.db import connection
+        q1 = len(connection.queries)
+        for ix in range(1):
+            container = self.create_container_with_samples(
+                prefix="container-{}".format(ix), sample_count=1)
+            container.save()
+        q2 = len(connection.queries)
```

Comment:
No assertion here.

Reply:
Yeah, this is a WIP test, I'll add a skip.

Reply:
Yes, skipping now as it's wip.